src/foam/nanos/crunch/Capability.js

```diff
@@ -330,8 +330,12 @@ foam.CLASS({
           if ( this.stringImplies(permissionName, permission) ) return true;
         }
 
+        // temporary prevent infinite loop when checking the permission "predicatedprerequisite.read.*"
+        // TODO : prerequisite checking below may be/probably is unnecessary
```

Comment:
todo dones in https://github.com/foam-framework/foam2/pull/5013

Reply:
left a comment in ^

Reply:
addressed in https://github.com/foam-framework/foam2/pull/5036